packages/institution-inspector/app/globals.css

```
@@ -0,0 +1,48 @@
+  --color-on-surface: #fdfbfe;
+  --color-on-surface-variant: #ababad;
+  --color-outline: #757578;
+  --color-outline-variant: #47484a;
+  --color-primary: #89acff;
+  --color-primary-dim: #5a90ff;
+  --color-secondary: #68fadd;
+  --color-tertiary: #e0f4ff;
+  --color-magenta: #ff2d8f;
+  --color-violet: #9b7bff;
+  --color-error: #ff716c;
+  --font-display: var(--font-manrope), Inter, ui-sans-serif, system-ui, sans-serif;
```

Stylelint (17.13.0)

[error] Expected "Inter" to be "inter" (value-keyword-case)

(value-keyword-case)